### ISSUE TITLE
`experimentalWebKitSupport`: Set `devicePixelRatio`

### ISSUE THREAD

**Issue by flotwig** [cypress-io/cypress#23808]
In other browsers, we set a standard `devicePixelRatio`. One reason for this is so that screenshot diffs can match whether a user runs them locally with 2x DPI or in CI with 1x DPI.

This also needs to be set in WebKit.

**Comment by bitcoinBandit1234** [comment]
It seems no one has taken this issue. can you please elaborate a bit on how you want the "devicePixelRatio" to be handled?

**Comment by nagash77** [comment]
Hi @bitcoinBandit1234 are you interested in picking up this ticket and opening a PR?

**Comment by bitcoinBandit1234** [comment]
@nagash77 yeah. but I need some more information on how this needs to be handled.
